sections/exchange/TradeCard/Charts/CombinedPriceChartCard.tsx

```diff
@@ -207,9 +217,12 @@ const CombinedPriceChartCard: FC<CombinedPriceChartCardProps> = ({
 							{PERIOD_LABELS.map((period) => (
 								<StyledTextButton
 									key={period.period}
-									isActive={period.period === selectedPeriod}
+									isActive={period.period === selectedChartPeriod}
+									disabled={
+										period.period !== Period.ONE_MONTH && isCandleStickChart && isOneMonthPeriod
```

Comment:
I don't think we should disable the other time periods. We should automatically switch the user back to the area chart when they click on any period other than 1M. 

We don't want people to get stuck and complain.

Reply:
Fixed.